libexec/python/docker/api.py

```diff
@@ -47,24 +52,46 @@ def create_runscript(cmd,base_dir):
     return output_file
 
 
-def get_token(repo_name,namespace="library",scope="repository",permission="pull"):
+def get_token(registry,repo_name,namespace,scope="repository",permission="pull",auth=None):
     '''get_token will use version 2.0 of Docker's service to return a token with given permission and scope - this
```

Comment:
I think it makes less sense here, I think functions like `get_config`, `get_manifest` or `get_layer` are part of the actual API of your module. But it would be weird to have `get_token` part of the public API, you always need a token for something :)
But I can change it for the sake of consistency if you want.